dali/operators/reader/loader/video_loader.cc

```diff
@@ -357,6 +357,15 @@ OpenFile& VideoLoader::get_or_open_file(const std::string &filename) {
       DALI_FAIL(err.str());
     }
     file.open = true;
+    recently_opened_.push_back(filename);
+    if (recently_opened_.size() > MAX_OPEN_FILE) {
+      // close the oldest file
+      auto& file_to_close = open_files_[recently_opened_.front()];
```

Comment:
Shouldn't you remove the entry from `open_files_` as well?

Reply:
No. It is enough to `file_to_close.open = false`.